packages/react/src/scroll-area/root/ScrollAreaRoot.tsx

```diff
@@ -35,10 +38,10 @@ const ScrollAreaRoot = React.forwardRef(function ScrollAreaRoot(
 
   const contextValue = React.useMemo(
     () => ({
-      dir,
+      direction,
```

Comment:
I am curious, what's the benefit of adding the direction in a context, when each subcomponent that need it can read the DirectionProvider's context?